check-shortlog/action.yml

```diff
@@ -19,6 +19,7 @@ runs:
         sed \
             -n \
             -e 's/ - //' \
+            -e 's/ (.*)//' \
```

Comment:
Note, if someone puts two spaces between the bullet point and their name it will get stripped, no issue, just worth knowing.